src/Symfony/Component/Routing/Generator/AstGenerator/GeneratorAstGeneratorInterface.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Routing\Generator\AstGenerator;
+
+use PhpParser\Node\Name;
+use PhpParser\Node\Param;
+use Symfony\Component\Ast\NodeList;
+
+/**
+ * @author Guilhem N. <egetick@gmail.com>
+ */
+interface GeneratorAstGeneratorInterface
```

Comment:
This interface looks unnecessary. You directly use the `GeneratorAstGenerator` class.